app/views/user_mailer/report_email.html.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Review Reporting!!!</h1>
+    <p>
+      Reported review for <br>
+      Review id : <%= @reviewid %> <br>
+      Professor id: <%= @review.professor_id %> <br>
+      Course id : <%= @review.course_id %> <br>
+      Semester : <%= @review.semester %> <br>
+      Year : <%= @review.year %>
+      
+    </p>
+    
+    <p>Please review this issue soon!!!!!</p>
+  </body>
```

Comment:
All of this information can be found using `@review`. In addition, none of the information presented here allows the administrator to take any action. They see a lot of information about the review (which, as an aside, can all be derived from the `@review` instance provided), but they don't know how to investigate it further or why it was reported in the first place, because the information from the review dialog isn't used at all.

Please include some sort of link to the review being reported, as well as the reasons why it was reported.